UPGRADING

```diff
@@ -50,6 +50,18 @@ PHP 8.5 UPGRADE NOTES
   . pcntl_exec() now throws ValueErrors when entries or keys of the
     $env_vars parameter contain null bytes.
 
+- PDO:
+  . The constructor arguments set in conjunction with PDO::FETCH_CLASS now
+    follow the usual CUFA semantics.
```

Comment:
Besides the fact that I think most people don't now what CUFA means, I think this PR is good to go.